snap/validate.go

```diff
@@ -414,6 +414,17 @@ func ValidateApp(app *AppInfo) error {
 		return err
 	}
 
+	// validate refresh-mode
+	switch app.RefreshMode {
+	case "", "restart", "survive":
```

Comment:
Could those be constans rather than literals?

Reply:
Maybe, we don't use constants for e.g. `app.Daemon` - but if you have strong opinions I can use constants here.